test/README.md

```diff
@@ -50,6 +50,8 @@ AWS_ACCESS_KEY_ID=XXX AWS_SECRET_ACCESS_KEY=xxx npm run integration-test-run-all
 
 _Note: Home folder is mocked for test run, therefore relying on `AWS_PROFILE` won't work. _ and _secret key_, need to be configured directly into env variables\_
 
+\_Note: This run test depending on shared infrastructure stack (see below)
```

Comment:
Can we put it with different PR (PR's are merged with single commits and ideally if no side, not relevant changes are included with the commit) ?

Let's fix the formatting, and reword it to: _Some integration tests depend on shared infrastructure stack (see below)_

Reply:
moved to #8867

Reply:
This has been merged.